dags/utils/onepassword.py

```diff
@@ -38,3 +40,12 @@ def get_item_by_title(entry_name):
     """
     client = get_client()
     return client.get_item_by_title(entry_name, VAULT_ID)
+
+
+@task(
+    task_id="get_env_vars",
+    execution_timeout=duration(seconds=30),
+)
+def get_env_vars_task(required_secrets):
+    """Airlfow task to fetch required secrets via load_dict"""
```

Comment:
this is to wrap the function in an airflow task to cut down on boilerplate, right? just wanting to make sure i understand correctly

also theres a typo in airflow

Reply:
yes and thank you! i have tried to improve the doctring.